fractions/monitor/src/main/java/org/wildfly/swarm/monitor/runtime/InstallMonitorFilter.java

```diff
@@ -47,10 +42,11 @@
 
     @Override
     public void prepareArchive(Archive<?> archive) {
-        JARArchive jarArchive = archive.as(JARArchive.class);
-
-        Asset resource = new ClassAsset(HealthResponseFilter.class);
-        ArchivePath location = new BasicPath(PATH_CLASSES, AssetUtil.getFullPathForClassResource(HealthResponseFilter.class));
-        jarArchive.add(resource, location);
+        try {
```

Comment:
So, did a quick check, and you're already calling the method I was thinking of `WARArchive.addDependency()`.

Did you try it with just GA? According to method that should work, and saves relying on version, etc.